tutorials/model_and_optimize_meshes.md

```diff
@@ -0,0 +1,139 @@
+
+@image html files/model_and_optimize_meshes/7_scale.jpg width=600px
+
+ - Bevel: allow you to create curved/chamfered edges
+This was used to create the curved sides of the wheel.
+
+@image html files/model_and_optimize_meshes/8_bevel.jpg width=600px
+
+- Extrude: adds additional geometry along a selected face/edges. The extrude tool can be used to push out a face or to push in one. Both examples are given here.
+
+@image html files/model_and_optimize_meshes/9_extrude.png width=600px
+
+This was used to create the indents of the wheel.
+
+`(Mesh > Extrude > Extrude Region)`
+
+ - Loop cut: allows you to split a face with a new looped edge. This then allows for more detailed manipulation of faces to take place.
+
+This with a combination of the extrude tool you can make small ridges to your shape.
+
+@image html files/model_and_optimize_meshes/10_loop_cut.jpg width=600px
+
+ - Duplicate: allows you to make identical copies of a selected object.
+
+@image html files/model_and_optimize_meshes/11_duplicate.jpg width=600px
+
+ - Other Modifiers:
+Some other commands that were used during a modeling process are: lattice functions,booleans, smooth, decimate.
+
+@image html files/model_and_optimize_meshes/12_other_modifiers.png width=600px
+
+#### More polygons less performance
+
+Shapes are made out of polygons. The higher the polygons on a shape the more detailed it is. A shape with too many polygons can overcomplicate mesh editing or can make simulations run very slowly. This is why it is important to optimize a shape so that it keeps most of its details but without making the poly count too high.
+
+@image html files/model_and_optimize_meshes/13_polygons.jpg "Left: Low Poly | Right: High Poly" width=600px
+
+
+If you find that your model is too dense to work with. The “decimate” command can be used to lower the polycount of said mesh, while still keeping its general shape.
+
+Creating a complex model requires a good understanding of many tools in a 3d modeling program. They all serve their specific purposes and are extremely powerful. They have to be used on top of each other in different ways to get to an end result. The more you add/edit your shape the better it will look in the long run.
+
+## Exporting models
+
+When you export a model for Gazebo, the .dae (COLLADA) file type is recommended.
+STL and OBJ files are also supported, but are less common for imports into Gazebo.
+
+To export: select your shape and then `File>Export>Collada(Default)(.dae)`
+
+@image html files/model_and_optimize_meshes/14_export_model.png width=600px
+
+After you hit export in whatever file format you decide to go with, the "Blender File View" will appear. This is where you choose the location of your file. You can also choose how you want to export your model.
+
+Double checking the “global Orientation” can save time and here you can quickly change the “Up Axis” to Y up. This is the standard orientation in Gazebo
```

Comment:
Gazebo uses Z up, I'm checking with @RafiAbdullah

Reply:
Confirmed that it should be Z up and updated in b121607d17c95701d6eb0531beb1884fefdce3ea